docs/hostname.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_label: Customize hostname
+title: ''
+---
+
+<head>
+  <link rel="canonical" href="https://elemental.docs.rancher.com/hostname"/>
+</head>
+
+### How to set a custom hostname
+
+When booting a host with an Elemental ISO, the hostname is temporarily set to `rancher-{$RANDOM}`.
```

Comment:
Same here as the other comment. 
And the PR I mentioned is this one: https://github.com/rancher/elemental/pull/1171/files

Reply:
The plan was to first document the actual state and how the hostname is managed, then update the next section.
For the next update I want to wait https://github.com/rancher/elemental/pull/1171 or some other change allowing hostname from DHCP is merged.
Before merging 1171 would like to try another approach  (i.e., set a transient hostname).